### PR TITLE
*: sync alias pages in translations

### PR DESCRIPTION
**Pull request by blueskyson**
Sync with `set-alias-page.py` from #6845 .

### CODE REVIEW

File: pages.ja/common/llvm-g++.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# llvm-g++
+
+> このコマンドは `clang++` のエイリアスです.
```
Comment (navarroaxel) [inline pages.ja/common/llvm-g++.md:3]:
```suggestion
> このコマンドは `clang++` のエイリアスです。
```

File: pages.ja/common/llvm-ar.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# llvm-ar
+
+> このコマンドは `ar` のエイリアスです.
```
Comment (navarroaxel) [inline pages.ja/common/llvm-ar.md:3]: Should this use this `dot`?
```suggestion
> このコマンドは `ar` のエイリアスです。
```

Comment (marchersimon) [reply pages.ja/common/llvm-ar.md:3]: Yes, the Japanese pages use the full-width period. But this should be fixed in [alias-pages.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md#ja) first.

**Comment by marchersimon** [comment]
There are a lot more Japanese alias pages with the wrong period, but I think we should finally merge this PR and fix this later.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**